string lookupType
environment_communication_msgs/Map lookupParameters
environment_communication_msgs/Map payLoad